test/images/volume/iscsi/VERSION

```diff
@@ -1 +1 @@
-1.0
+2.0
```

Comment:
Whats there in `block.tar.gz` file jan?

Reply:
See README.md:

> block.tar.gz is a small ext2 filesystem created by `create_block.sh` (run as root!)

I had to update it from "Hello from iSCSI" to "Hello from iscsi" for a mount test to pass, as it expects test driver name there.